### ISSUE TITLE
Does not work with Cycles stereoscopic rendering

### ISSUE THREAD

**Issue by maro1234567** [neverhood311/Stop-motion-OBJ#130]
**System Information**
Operating system: Win10 64 bit

**Addon version**
v2.1.1

**Describe the bug**
Animated parts in the OBJ sequence to only render for one eye in stereoscopic rendering. 

**Steps To Reproduce**
1. install&enable stop-motion-obj and create scene with OBJ sequence
2. Change render engine to cycles
3. Enable stereoscopic rendering in the scene settings
4. Render
5. Animated parts will only render for one eye

**Expected behavior**
...need to render for both eyes ie cyan/magenta

**Screenshots**
![2021-06-22 13_22_26-Blender Render](https://user-images.githubusercontent.com/86297204/122916306-06891b00-d35d-11eb-9e86-f8bbcea68b57.jpg)

**Comment by neverhood311** [comment]
Thanks for letting me know. I'll get to it when I can, but I don't anticipate having much time in the near future.

I've been looking for workarounds that might help you get by in the meantime. Here's an idea (it's a bit of a pain, but it will probably get the job done). You could create your own stereo camera rig with two regular cameras, grouped together. Render the animation twice: first through the left camera and then through the right camera, saved to separate image sequences. Finally, open a new Blender project and use the general idea from [this Photoshop tutorial](https://photography.tutsplus.com/tutorials/how-to-turn-your-photos-into-anaglyph-3d-images--photo-5776) to help you combine the two image sequences using the Compositor. I think you should be able to use Blender's Compositor nodes to achieve the same effect.

**Comment by maro1234567** [comment]
Thanks for the answer! Your suggested workaround (and dependent on the case - rendering monoscopic sprites/billboards first) is what I already started to do for the project I currently try to finish, nevertheless I wanted to report the issue.

**Comment by neverhood311** [comment]
@maro1234567 I'm looking into this now. 

Have you tried stereoscopic rendering recently? Using the development version of the addon (2.2.0.alpha.16) and Blender 2.93.3, I'm not experiencing this problem. I've tried both Cycles and Eevee. I've also tried rendering into a single image per frame (Red-Cyan Anaglyph) as well as individual images (e.g. `frame01_L.png, frame01_R.png`). I even tried both Cached and Streaming sequences.

No issues in any case.

Hopefully you experience the same now.
